scripts/fetch-entity/src/index.ts

```diff
@@ -50,76 +27,21 @@ program
     'Limit the number of resources to fetch (only with --collection)',
     '1'
   )
-  .action(
```

Comment:
Moved this into a separate module because I was at one point trying to load the TS file for this directly in the mcp server and call the function directly. I ended up not doing that in the end, but it seemed like an okay thing to separate into its own module to separate the command-line interface from the business logic of the tool itself.